lib/urlwatch/reporters.py

```diff
@@ -65,6 +66,11 @@
 except ImportError:
     Markdown = None
 
+try:
+    import aioxmpp
```

Comment:
This dependency should be mentioned in the documentation (`docs/source/dependencies.rst`).